cypress/integration/save.cy.js

```diff
@@ -194,6 +194,13 @@ describe('save', () => {
         expectAOTitleToContainExact(AO_NAME)
         expectTableToBeVisible()
 
+        cy.url()
+            .should('match', /\/[a-zA-Z][a-zA-Z0-9]{10}$/)
```

Comment:
This regexp is repeated many times, we could extract it and reuse it.

Reply:
`const uidRe = /\/[a-zA-Z][a-zA-Z0-9]{10}$/` should be sufficient.

Reply:
Done